### PR TITLE
Refs #7: Add multiple choices form field.

### PR DESCRIPTION
**Pull request by n1k0**
Refs #7, cc @almet

### CODE REVIEW

File: formbuilder/config.js [hunk 1]
```diff
@@ -72,6 +72,39 @@ export default {
...
+              title: "Choices",
+              items: {
+                type: "string"
+              },
+              default: ["choice 1", "choice 2", "choice 3"],
+            }
+          }
+        },
+      },
+      formData: {}
```
Comment (almet) [inline formbuilder/config.js]: And of course when you read it it all looks so simple. If only I hadn't tried :-)

File: formbuilder/components/builder/DescriptionField.js [hunk 2]
```diff
@@ -6,8 +6,17 @@ function DescriptionField(props) {
...
+  const {id, description=""} = props;
+  return (
+    <p id={id}>
+      <RIEInput
+        className="edit-in-place"
+        classEditing="edit-in-place-active"
+        propName="description"
+        value={description}
+        change={onUpdate} />
+    </p>
```
Comment (almet) [inline formbuilder/components/builder/DescriptionField.js]: Better :-)

**Comment by almet** [comment]
Okay, they're actually failing on master, so not related.

**Comment by almet** [comment]
Actually, this seems related to a compatibility change with RJSF, which doesn't use the `ui:order` property the same way it did previously.

**Comment by almet** [comment]
I've fixed one of the two failing tests, but I don't undestand why the second is failing. @n1k0 do you have any idea?

**Comment by n1k0** [comment]
Nope but I indeed suspect an issue with the lib version bumps here. I'll investigate.

**Comment by almet** [comment]
I've seen problems like this in the past and was also stuck. Thanks!

**Comment by n1k0** [comment]
@almet should be all good now.

**Pull request merged**